src/components/dropdown/dropdown.scss

```diff
@@ -16,6 +16,10 @@ $dropdown-input-icon-width: 20px !default;
   }
 }
 
+.carbon-dropdown__list-hidden {
+  display: none;
```

Comment:
Are we sure about `display: none` - it also removes the things from the DOM

Will automation ever process css? Also, are their accessibility considerations here?